packages/babel-types/test/fields.js

```diff
@@ -52,6 +52,26 @@ const ignoredFields = {
   ObjectMethod: { method: true, id: true, predicate: true },
   StaticBlock: { static: true },
   TSDeclareMethod: { id: true },
+  // TODO(Babel 8): Remove
+  // https://github.com/babel/babel/issues/9231
+  ...(process.env.BABEL_8_BREAKING
```

Comment:
```suggestion
  ...(process.env.BABEL_8_BREAKING
```

No need for the comment, it's implicit in the `process.env.BABEL_8_BREAKING` check.